userguide/jails.rst

```diff
@@ -93,7 +93,8 @@ Notes about the :file:`iocage/` dataset:
 * :file:`iocage/` datasets on activated pools are independent of each
   other and do **not** share any data.
 
-.. note:: The iocage config is backed up to
+.. note:: iocage jail configs are stored in
+   :file:`/mnt/{poolname}/iocage/jails/{jailname}`. These are backed up to
```

Comment:
This is accurate, however they are not backed up to any BE. Not sure i follow that ?

Reply:
So it is accurate enough to remove that bit and just say 
```
iocage jail configs are stored in
:file:`/mnt/{poolname}/iocage/jails/{jailname}`. This means that rolling back
to a previous boot environment preserves jail and plugin settings.
```

Reply:
So that is accurate but we should clarify. The configs aren’t stored in BE, rolling back BE would leave us with working iocage jails, however the configs would have to be manually restored.

Reply:
ahh okay. Could you explain how to manually restore the config so I can properly document it for users?

Reply:
@a-stjohn so each time iocage updates the configuration file, existing configuration file ( `/mnt/{poolname}/iocage/jails/{jailname}/config.json`) is backed up like this `/mnt/{poolname}/iocage/jails/{jailname}/config_backup.json`. In order for the user to restore this, he/she needs to change the filename of `config_backup.json` to `config.json` ( preferably backing up `config.json` itself before doing this but of course it's up to them. If they intend to not save it, so be it, on an older version, it won't be usable anyways

Reply:
Awesome! Thank you!